src/pages/PointOfContact/index.tsx

```diff
@@ -217,7 +217,7 @@ function PointOfContact(): JSX.Element {
         <FormHeaderWrapper>
           <TextIntroduction
             heading='Provide point of contact'
-            subheading="Provide the name and business contact information of a person that the Bureau or other regulators may contact with questions about your financial institution's data submission."
+            subheading="Provide the name and business contact information of a person that the Bureau or other regulators may contact with questions about your financial institution's data filing."
```

Comment:
@billhimmelsbach - Remove the word "data" before "filing"

Reply:
[Changed](https://github.com/cfpb/sbl-frontend/pull/883/commits/31314769428e86d5075de45c9da48cf9499344c8)!